OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -312,7 +312,8 @@ public MovePart(Move move, WPos from, WPos to, int fromFacing, int toFacing, int
 				MoveFractionTotal = (to - from).Length;
 
 				// Calculate an elliptical arc that joins from and to
-				if (fromFacing != toFacing)
+				var delta = Util.NormalizeFacing(fromFacing - toFacing);
+				if (delta != 0 && delta != 128)
```

Comment:
No, that would make it run unconditionally.  If a number is == 0 then it certainly != 128, and vice versa.